x-pack/plugins/security_solution/public/flyout/document_details/right/footer.tsx

```diff
@@ -182,8 +178,13 @@ export const PanelFooter: FC<PanelFooterProps> = ({ isPreview }) => {
 
   return (
     <>
-      <StyledEuiFlyoutFooter data-test-subj={FLYOUT_FOOTER_TEST_ID}>
-        <EuiFlexGroup justifyContent="flexEnd">
+      <EuiFlyoutFooter data-test-subj={FLYOUT_FOOTER_TEST_ID} className="side-panel-flyout-footer">
+        <EuiFlexGroup
```

Comment:
also instead of having some custom css here, what do you think about wrapping the `EuiFlexGoup` with an ```<EuiPanel color="transparent">...</EuiPanel>```
The UI would look exactly the same and we don't use custom css, just plain EUI components?

Reply:
custom css is used throughout our code base, i do admit it doesn't look good. I prefer it over eui panel because it's more explicit on what we are changing. Replaced with EuiPanel in the [latest commit](1e9dc053815324a30594302d9886f64efbe9c3b2)